packages/app-mobile/components/voiceTyping/AudioRecordingBanner.tsx

```diff
@@ -119,6 +119,11 @@ const useAudioRecorder = (onFileSaved: OnFileSavedCallback, onDismiss: ()=> void
 				if (!response.granted) {
 					throw new Error(_('Missing permission to record audio.'));
 				}
+
+				// Work around "This experience is currently in the background, so the audio session could not be activated"
+				// See https://github.com/expo/expo/issues/21782
+				// May be resolved by migrating to expo-audio.
+				await msleep(500);
```

Comment:
It may make sense to move this change to a separate pull request, as this is not a regression related to the React Native upgrade. See #12314.